byterun/obj.c

```diff
@@ -150,13 +150,34 @@ CAMLprim value caml_obj_compare_and_swap (value v, value f, value oldv, value ne
   return Val_int(res);
 }
 
-/* caml_promote_to(obj, upto) promotes obj to be as least as shared as upto */
+/* caml_promote_to(obj, upto) may promote obj to be as least as shared as upto.
+   Skips promotion if there is only a single domain. This should only be used in
+   situations where there is a performance need to 'front-load' the promotion
+   across domains, such as for queues. If you need to guarantee promotion to the
+   major heap for correctness, */
```

Comment:
I think half of this sentence got lost somewhere!

Reply:
Whoops. I'm really not sure where that went. Fixed.